src/oic/utils/keyio.py

```diff
@@ -396,6 +407,7 @@ def __init__(self, ca_certs=None, verify_ssl=True, keybundle_cls=KeyBundle):
         self.ca_certs = ca_certs
         self.verify_ssl = verify_ssl
         self.keybundle_cls = keybundle_cls
+        self.remove_after = remove_stale_after
```

Comment:
By choice but I guess mostly of historic reasons. 
So should possibly make them the same.